enumflags_derive/src/lib.rs

```diff
@@ -181,21 +182,21 @@ fn gen_enumflags(ident: &Ident, item: &DeriveInput, data: &DataEnum) -> TokenStr
                        fmt: &mut #std_path::fmt::Formatter)
                        -> #std_path::fmt::Result {
                     use ::enumflags2::RawBitFlags;
-                    let v:Vec<&str> =
-                        [#((#names :: #variants).bits(),)*]
-                        .iter()
-                        .filter_map(|val|{
-                            let val: #ty = *val as #ty & flags.bits();
-                            match val {
-                                #(#flag_value_names => Some(stringify!(#variants)),)*
-                                _ => None
-                            }
-                        })
-                        .collect();
-                    write!(fmt, "BitFlags<{}>(0b{:b}, [{}]) ",
+                    use #std_path::iter::Iterator;
```

Comment:
Yeah this one is an example of code that could be pulled into main crate, whether via `_internal` and re-exported or by changing the `RawBitFlags` trait to include `flag_names` alongside `flag_list` (then throw this logic directly into the `Debug` impl). Either approach is an intrusive/breaking change so is left out of this PR, though between the two approached I think changing the trait would be the cleaner solution.

Reply:
Okay. I've made a decision about the breaking changes. The next release is going to be 0.6.0, because the minimal rustc version was bumped by syn. I do agree that changing the trait is the cleaner solution. If you want to, you can do this, but I won't mind if you don't -- I'll merge this as-is and refactor it later.

Reply:
Ah alright, well it does still feel worth separating them regardless. My concern about being able to update 0.5.x was mostly just due to wanting to make the `nostd` feature flag work, but I'm in no rush and can wait for 0.6

Reply:
The changes for this project are infrequent enough that the major version bump isn't going to make me change the release timing, so don't worry.